File: Unidad 3/JAVA/P12_U3.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.p12_u3;

import java.util.Scanner;

/**
 *
 * @author LaboratorioLTI2
 */
public class P12_U3 {

    public static void main(String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int c, total, num;
        System.out.println("Escribe el total de número a procesar");
        total = sc.nextInt();
        
        for (c=1; c<=total; c++) {
            System.out.println("Escribe el número:"); 
            num = sc.nextInt();
            
            if ( num %2 ==0){
                System.out.println("Número PAR");
            }
            else {
                System.out.println("Número IMPAR");
            }
        }
        
    }
}

File: Unidad 3/C#/P12_U3.cs
using System;

namespace MyApp
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int c, total, num;
            Console.WriteLine("Escribe el total de número a procesar");
            total = Convert.ToInt32(Console.ReadLine());

            for (c = 1; c <= total; c++)
            {
                Console.WriteLine("Escribe el número:");
                num = Convert.ToInt32(Console.ReadLine());

                if (num % 2 == 0)
                {
                    Console.WriteLine("Número PAR");
                }
                else
                {
                    Console.WriteLine("Número IMPAR");
                }
            }
        }
    }
}